cmd/jb/main.go

```diff
@@ -45,6 +45,11 @@ var (
 		initActionName,
 		installActionName,
 	}
+	gitSSHRegex                   = regexp.MustCompile("git\\+ssh://git@([^:]+):([^/]+)/([^/]+).git")
```

Comment:
Unlike original prefixes, these provide a url scheme. I think as a convenience, it's nice for the `https://` to be implicit with github.com despite it inconsistent.